assets/xml/objects/object_bg.xml

```diff
@@ -19,44 +22,58 @@
+        <Limb Name="gTreasureChestShopGalHeadLimb" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_HEAD" Offset="0x8E88" />
+        <Limb Name="gTreasureChestShopGal_008E94" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_06" Offset="0x8E94" />
+        <Limb Name="gTreasureChestShopGalLeftUpperArmLimb" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_LEFT_UPPER_ARM" Offset="0x8EA0" />
+        <Limb Name="gTreasureChestShopGalLeftForearmLimb" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_LEFT_FOREARM" Offset="0x8EAC" />
+        <Limb Name="gTreasureChestShopGalLeftHandLimb" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_LEFT_HAND" Offset="0x8EB8" />
+        <Limb Name="gTreasureChestShopGal_008EC4" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_0A" Offset="0x8EC4" />
+        <Limb Name="gTreasureChestShopGalRightUpperArmLimb" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_RIGHT_UPPER_ARM" Offset="0x8ED0" />
+        <Limb Name="gTreasureChestShopGalRightForearmLimb" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_RIGHT_FOREARM" Offset="0x8EDC" />
+        <Limb Name="gTreasureChestShopGalRightHandLimb" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_RIGHT_HAND" Offset="0x8EE8" />
+        <Limb Name="gTreasureChestShopGal_008EF4" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_0E" Offset="0x8EF4" />
+        <Limb Name="gTreasureChestShopGalRightThighLimb" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_RIGHT_THIGH" Offset="0x8F00" />
+        <Limb Name="gTreasureChestShopGalRightCalfLimb" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_RIGHT_CALF" Offset="0x8F0C" />
+        <Limb Name="gTreasureChestShopGal_008F18" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_11" Offset="0x8F18" />
+        <Limb Name="gTreasureChestShopGalRightFootCalfLimb" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_RIGHT_FOOT_CALF" Offset="0x8F24" />
+        <Limb Name="gTreasureChestShopGal_008F30" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_13" Offset="0x8F30" />
+        <Limb Name="gTreasureChestShopGalLeftThighLimb" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_LEFT_THIGH" Offset="0x8F3C" />
+        <Limb Name="gTreasureChestShopGalLeftCalfLimb" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_LEFT_CALF" Offset="0x8F48" />
+        <Limb Name="gTreasureChestShopGal_008F54" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_16" Offset="0x8F54" />
+        <Limb Name="gTreasureChestShopGalLeftFootLimb" Type="Standard" EnumName="TREASURE_CHEST_SHOP_GAL_LIMB_LEFT_FOOT" Offset="0x8F60" />
+        
+        <!-- Treasure Chest Shop Gal Skeleton -->
+        <Skeleton Name="gTreasureChestShopGalSkeleton" Type="Flex" LimbType="Standard" LimbNone="TREASURE_CHEST_SHOP_GAL_LIMB_NONE" LimbMax="TREASURE_CHEST_SHOP_GAL_LIMB_MAX" EnumName="ObjectBg1Limb" Offset="0x8FC8" />
```

Comment:
Skeletons should end in `Skel`, not `Skeleton`. You could also name the `EnumName` while you're here.